src/bench/index_blockfilter.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2023-present The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+#include <bench/bench.h>
```

Comment:
Nit: Put this include on its own line (like e.g. in `bip324_ecdh.cpp`), or fix the include order.

Reply:
done